File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/comum/validators/calculo/Alerta.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.comum.validators.calculo;

import br.jus.trt8.pjecalc.negocio.comum.validators.calculo.MsgValidador;

public class Alerta
extends MsgValidador {
    private static final long serialVersionUID = -5647914041529242456L;
    private static final String TIPO = "ALERTA";

    @Override
    public boolean isImpeditivo() {
        return false;
    }

    @Override
    public String getTipo() {
        return TIPO;
    }
}

File: src/lib/pjecalc/core/comum/validators/calculo/alerta.ts
/**
 * PJe-Calc v2.15.1 — Alerta
 * Porte de: br.jus.trt8.pjecalc.negocio.comum.validators.calculo.Alerta
 */
import { MsgValidador } from './msg-validador';

export class Alerta extends MsgValidador {
  private static readonly TIPO = 'ALERTA';
  isImpeditivo(): boolean { return false; }
  getTipo(): string { return Alerta.TIPO; }
}
